src/adagrams.js

```diff
@@ -1,8 +1,140 @@
-const Adagrams = {
-  drawLetters() {
-    // Implement this method for wave 1
-  },
+class Adagrams {
+  static drawLetters() {
+    const letterValues = {
+      A: 9,
+      B: 2,
+      C: 2,
+      D: 4,
+      E: 12,
+      F: 2,
+      G: 3,
+      H: 2,
+      I: 9,
+      J: 1,
+      K: 1,
+      L: 4,
+      M: 2,
+      N: 6,
+      O: 8,
+      P: 2,
+      Q: 1,
+      R: 6,
+      S: 4,
+      T: 6,
+      U: 4,
+      V: 2,
+      W: 2,
+      X: 1,
+      Y: 2,
+      Z: 1,
+    };
+    //make an array containing all available letters 
+    let sample = Object.entries(letterValues).map(([key , value]) => {
+      return key.repeat(value).split('');
+    });
+
+    sample = sample.flat();
+
+    const lettersAtHand = [];
+
+    for (let i = 0; i < 10; i++){
```

Comment:
Good use of `Math.random()` to select a random letter. However, what keeps you from choosing a Z 10 times (however unlikely)?